src/state/lists/reducer.ts

```diff
@@ -47,10 +47,12 @@ const initialState: ListsState = {
 export default createReducer(initialState, (builder) =>
   builder
     .addCase(fetchTokenList.pending, (state, { payload: { requestId, url } }) => {
+      const current = state.byUrl[url]?.current ?? null
+      const pendingUpdate = state.byUrl[url]?.pendingUpdate ?? null
+
       state.byUrl[url] = {
-        current: null,
-        pendingUpdate: null,
-        ...state.byUrl[url],
+        current: current,
```

Comment:
can you shorten to this?

```
{
current,
pendingUpdate,
loadingRequestId: requestId,
error: null,
}
```

Reply:
I expect this to be covered by eslint. Can I just make a separate PR where I set the object-shorthand rule to "always", and fix the errors (there's a few files).

Reply:
I could do it as a new commit on this PR, but it affects 7 files, and I'd rather avoid the scope creep.